src/libraries/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/SymmetricAlgorithm.cs

```diff
@@ -980,6 +980,481 @@ public byte[] EncryptCbc(byte[] plaintext, byte[] iv, PaddingMode paddingMode =
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///   <para>
+        ///     <paramref name="paddingMode" /> is not a valid padding mode.
+        ///   </para>
+        ///   <para>
+        ///     -or-
+        ///   </para>
+        ///   <para>
+        ///     <paramref name="feedbackSizeInBits" /> is not positive or represent a whole number of bytes.
+        ///   </para>
+        /// </exception>
+        /// <exception cref="ArgumentException">
+        ///   <para>
+        ///     <paramref name="iv" /> is the incorrect length. Callers are expected to pass an initialization vector
+        ///     that is exactly <see cref="BlockSize" /> in length, converted to bytes (<c>BlockSize / 8</c>).
+        ///   </para>
+        ///   <para>
+        ///     -or-
+        ///   </para>
+        ///   <para>
+        ///     The buffer in <paramref name="destination"/> is too small to hold the plaintext data.
+        ///   </para>
+        /// </exception>
+        /// <exception cref="CryptographicException">
+        ///   <para>
+        ///     The ciphertext could not be decrypted successfully.
+        ///   </para>
+        ///   <para>
+        ///     -or-
+        ///   </para>
+        ///   <para>
+        ///     <paramref name="feedbackSizeInBits" /> is not valid for the algorithm.
+        ///   </para>
+        /// </exception>
+        /// <remarks>
+        ///   This method's behavior is defined by <see cref="TryDecryptCfbCore" />.
+        /// </remarks>
+        public int DecryptCfb(
+            ReadOnlySpan<byte> ciphertext,
+            ReadOnlySpan<byte> iv,
+            Span<byte> destination,
+            PaddingMode paddingMode = PaddingMode.None,
+            int feedbackSizeInBits = 8)
+        {
+            CheckPaddingMode(paddingMode);
+            CheckInitializationVectorSize(iv);
+            CheckFeedbackSize(feedbackSizeInBits);
+
+            if (!TryDecryptCfbCore(ciphertext, iv, destination, paddingMode, feedbackSizeInBits, out int written))
+            {
+                throw new ArgumentException(SR.Argument_DestinationTooShort, nameof(destination));
+            }
+
+            return written;
+        }
+
+        /// <summary>
+        ///   Attempts to decrypt data into the specified buffer, using CFB mode
+        ///   with the specified padding mode and feedback size.
+        /// </summary>
+        /// <param name="ciphertext">The data to decrypt.</param>
+        /// <param name="iv">The initialization vector.</param>
+        /// <param name="destination">The buffer to receive the plaintext data.</param>
+        /// <param name="paddingMode">The padding mode used to produce the ciphertext and remove during decryption.</param>
+        /// <param name="feedbackSizeInBits">The feedback size, specified in bits.</param>
+        /// <param name="bytesWritten">When this method returns, the total number of bytes written to <paramref name="destination" />.</param>
```

Comment:
Canonically, this would be up two lines (matching the order of declaration).  But it doesn't technically matter.

Reply:
👍 Fixed a few xml doc orderings.